trigger/src/main/scala/core/persistance/Repository.scala

```diff
@@ -0,0 +1,15 @@
+package core.persistance
+
+import core.models.tables._
+import slick.jdbc.MySQLProfile.api._
+
+trait Repository {
+  val db = Database.forConfig("mysqlDB")
```

Comment:
parameterizable?

Reply:
I am not sure about this one. This literal is used only once on this place.

Reply:
Maybe I'm not getting it. How would we change it to Postgres, for instance?

Reply:
It's going to be more complicated. You have to replace all "import slick.jdbc.MySQLProfile.api._
". 
I will replace "mysqlDB" with "dbConfig" literal and I will extract all "import slick.jdbc.MySQLProfile.api._" into interface so there should be only once place where is this import.

Reply:
Refactored: 
1. "mysqlDB" -> "db"
2. imports slick.jdbc.MySQLProfile.api._ replaced with import core.models.tables.JDBCProfile.profile._

If you want to change DB you have to replace db properties in prop. file and change core.models.tables.JDBCProfile class and use correct profile.